src/ASM/ASMs2DSpec.C

```diff
@@ -307,6 +307,7 @@ bool ASMs2DSpec::evalSolution (Matrix& sField, const IntegrandBase& integrand,
 
 	// Compute the Jacobian inverse
 	fe.detJxW = utl::Jacobian(Jac,fe.dNdX,Xnod,dNdu);
+        if (fe.detJxW == 0.0) continue; // skip singular points
```

Comment:
Wrong indent

Reply:
it's not, the indents are correct, the surrounding code uses tabs..